docs/_user_guide/best_practices_matlab_slurm_job.md

```diff
@@ -1,33 +1,33 @@
 ## Matlab Example Slurm Script
 The simplest Matlab script looks like:
 
-```matlab
+```matlab title="Matlab Code"
 disp('Hello world')
 ```
 
 Save this line to a new file called `hello_world.m`. 
 
-## Available Matlab versions on Yen10
+## Available Matlab versions on `Yen10`
```

Comment:
remove this entire section line, I'm not sure this page actually needs any subsections